src/main/webapp/app/videosource/gocast/gocast-course-binding.component.html

```
@@ -0,0 +1,79 @@
+<p-card styleClass="mt-3">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)